openedx_ledger/models.py

```diff
@@ -716,8 +716,9 @@ class Deposit(TimeStampedModelWithUuid):
     )
     sales_contract_reference_id = models.CharField(
         max_length=255,
-        blank=False,
-        null=False,
+        # Don't require this field because it's often inherited from the parent subsidy which also has blank/null=True.
+        blank=True,
+        null=True,
```

Comment:
[sanity check / clarification] Does the `sales_contract_reference_provider` field below also need to made optional if this `sales_contract_reference_id` field is now optional? If `sales_contract_reference_id` is NULL, what would the value of `sales_contract_reference_provider` be?